2019-07-01 11:18:40,505 ERROR: Exception on / [GET] [in /anaconda3/lib/python3.6/site-packages/flask/app.py:1761]
Traceback (most recent call last):
  File "/anaconda3/lib/python3.6/site-packages/flask/app.py", line 2292, in wsgi_app
    response = self.full_dispatch_request()
  File "/anaconda3/lib/python3.6/site-packages/flask/app.py", line 1815, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/anaconda3/lib/python3.6/site-packages/flask/app.py", line 1718, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/anaconda3/lib/python3.6/site-packages/flask/_compat.py", line 35, in reraise
    raise value
  File "/anaconda3/lib/python3.6/site-packages/flask/app.py", line 1813, in full_dispatch_request
    rv = self.dispatch_request()
  File "/anaconda3/lib/python3.6/site-packages/flask/app.py", line 1799, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/Users/christenkinney/Desktop/microblog/app/routes.py", line 35, in index
    posts = current_user.followed_posts().paginate(page, app.config['POSTS_PER_PAGE'], False)
  File "/anaconda3/lib/python3.6/site-packages/werkzeug/local.py", line 347, in __getattr__
    return getattr(self._get_current_object(), name)
AttributeError: 'AnonymousUserMixin' object has no attribute 'followed_posts'
2019-07-01 11:18:42,914 ERROR: Exception on /index [GET] [in /anaconda3/lib/python3.6/site-packages/flask/app.py:1761]
Traceback (most recent call last):
  File "/anaconda3/lib/python3.6/site-packages/flask/app.py", line 2292, in wsgi_app
    response = self.full_dispatch_request()
  File "/anaconda3/lib/python3.6/site-packages/flask/app.py", line 1815, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/anaconda3/lib/python3.6/site-packages/flask/app.py", line 1718, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/anaconda3/lib/python3.6/site-packages/flask/_compat.py", line 35, in reraise
    raise value
  File "/anaconda3/lib/python3.6/site-packages/flask/app.py", line 1813, in full_dispatch_request
    rv = self.dispatch_request()
  File "/anaconda3/lib/python3.6/site-packages/flask/app.py", line 1799, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/Users/christenkinney/Desktop/microblog/app/routes.py", line 35, in index
    posts = current_user.followed_posts().paginate(page, app.config['POSTS_PER_PAGE'], False)
  File "/anaconda3/lib/python3.6/site-packages/werkzeug/local.py", line 347, in __getattr__
    return getattr(self._get_current_object(), name)
AttributeError: 'AnonymousUserMixin' object has no attribute 'followed_posts'
2019-07-01 11:26:00,566 INFO: Microblog startup [in /Users/christenkinney/Desktop/microblog/app/__init__.py:56]
2019-07-01 11:26:09,517 ERROR: Exception on / [GET] [in /anaconda3/lib/python3.6/site-packages/flask/app.py:1761]
Traceback (most recent call last):
  File "/anaconda3/lib/python3.6/site-packages/flask/app.py", line 2292, in wsgi_app
    response = self.full_dispatch_request()
  File "/anaconda3/lib/python3.6/site-packages/flask/app.py", line 1815, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/anaconda3/lib/python3.6/site-packages/flask/app.py", line 1718, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/anaconda3/lib/python3.6/site-packages/flask/_compat.py", line 35, in reraise
    raise value
  File "/anaconda3/lib/python3.6/site-packages/flask/app.py", line 1813, in full_dispatch_request
    rv = self.dispatch_request()
  File "/anaconda3/lib/python3.6/site-packages/flask/app.py", line 1799, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/Users/christenkinney/Desktop/microblog/app/routes.py", line 35, in index
    posts = current_user.followed_posts().paginate(page, app.config['POSTS_PER_PAGE'], False)
  File "/anaconda3/lib/python3.6/site-packages/werkzeug/local.py", line 347, in __getattr__
    return getattr(self._get_current_object(), name)
AttributeError: 'AnonymousUserMixin' object has no attribute 'followed_posts'
2019-07-01 11:27:53,557 INFO: Microblog startup [in /Users/christenkinney/Desktop/microblog/app/__init__.py:56]
2019-07-01 12:50:44,253 INFO: Microblog startup [in /Users/christenkinney/Desktop/microblog/app/__init__.py:56]
2019-07-01 12:52:30,167 INFO: Microblog startup [in /Users/christenkinney/Desktop/microblog/app/__init__.py:56]
2019-07-01 12:53:17,457 INFO: Microblog startup [in /Users/christenkinney/Desktop/microblog/app/__init__.py:56]
2019-07-01 12:59:19,490 INFO: Microblog startup [in /Users/christenkinney/Desktop/microblog/app/__init__.py:56]
2019-07-01 12:59:50,679 INFO: Microblog startup [in /Users/christenkinney/Desktop/microblog/app/__init__.py:56]
2019-07-01 13:05:29,101 INFO: Microblog startup [in /Users/christenkinney/Desktop/microblog/app/__init__.py:56]
2019-07-01 13:06:27,836 INFO: Microblog startup [in /Users/christenkinney/Desktop/microblog/app/__init__.py:56]
2019-07-01 13:07:55,616 INFO: Microblog startup [in /Users/christenkinney/Desktop/microblog/app/__init__.py:56]
2019-07-01 13:10:10,298 INFO: Microblog startup [in /Users/christenkinney/Desktop/microblog/app/__init__.py:56]
2019-07-01 13:13:45,493 INFO: Microblog startup [in /Users/christenkinney/Desktop/microblog/app/__init__.py:56]
2019-07-01 13:15:26,941 INFO: Microblog startup [in /Users/christenkinney/Desktop/microblog/app/__init__.py:56]
2019-07-01 13:41:39,576 INFO: Microblog startup [in /Users/christenkinney/Desktop/microblog/app/__init__.py:56]
2019-07-01 14:04:53,878 INFO: Microblog startup [in /Users/christenkinney/Desktop/microblog/app/__init__.py:58]
2019-07-01 14:47:07,802 INFO: Microblog startup [in /Users/christenkinney/Desktop/microblog/app/__init__.py:59]
2019-07-08 07:33:17,475 INFO: Microblog startup [in /Users/christenkinney/Desktop/microblog/app/__init__.py:59]
2019-07-08 08:29:15,965 INFO: Microblog startup [in /Users/christenkinney/Desktop/microblog/app/__init__.py:61]
2019-07-08 09:16:42,935 INFO: Microblog startup [in /Users/christenkinney/Desktop/microblog/app/__init__.py:63]
2019-07-08 11:26:55,464 INFO: Microblog startup [in /Users/christenkinney/Desktop/microblog/app/__init__.py:77]
2019-07-08 11:39:53,399 INFO: Microblog startup [in /Users/christenkinney/Desktop/microblog/app/__init__.py:77]
2019-07-08 11:41:35,436 INFO: Microblog startup [in /Users/christenkinney/Desktop/microblog/app/__init__.py:77]
2019-07-08 12:22:17,111 INFO: Microblog startup [in /Users/christenkinney/Desktop/microblog/app/__init__.py:77]
2019-07-08 12:22:33,401 INFO: Microblog startup [in /Users/christenkinney/Desktop/microblog/app/__init__.py:77]
2019-07-09 09:03:29,795 INFO: Microblog startup [in /Users/christenkinney/Desktop/microblog/app/__init__.py:77]
2019-07-09 09:05:02,834 INFO: Microblog startup [in /Users/christenkinney/Desktop/microblog/app/__init__.py:77]
2019-07-09 09:05:02,834 INFO: Microblog startup [in /Users/christenkinney/Desktop/microblog/app/__init__.py:77]
2019-07-09 09:32:47,649 INFO: Microblog startup [in /Users/christenkinney/Desktop/microblog/app/__init__.py:77]
